adafruit_hue.py

```diff
@@ -130,14 +140,14 @@ def register_username(self):
         return username
 
     # Lights API
-    def show_light_info(self, light_id):
+    def show_light_info(self, light_id: Union[int, str]) -> str:
         """Gets the attributes and state of a given light.
         :param int light_id: Light identifier.
```

Comment:
Nice typing, `str` and `int` are both allowed!  Mind updating this `:param:` typing from `int` to `int|str`?